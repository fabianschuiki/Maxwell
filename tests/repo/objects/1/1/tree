{"inputs":{"class":"FunctionArgumentTuple"},"outputs":{"class":"FunctionArgumentTuple"},"body":{"class":"FunctionBody","stmts":[{"class":"ExprStmt","expr":{"class":"AssignmentExpr","lhs":{"class":"IdentifierExpr"},"rhs":{"class":"BinaryOperatorExpr","lhs":{"class":"IdentifierExpr"},"rhs":{"class":"ConstantExpr"}}}}]}}