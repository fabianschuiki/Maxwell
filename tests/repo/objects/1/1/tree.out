{"inputs":{"class":"FunctionArgumentTuple"},"outputs":{"class":"FunctionArgumentTuple"},"body":{"stmts":[{"class":"ExprStmt"}],"class":"FunctionBody"}}